src/index.html

```diff
@@ -1,13 +1,395 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
+    <title>Air</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="icon"
+      type="images/x-icon"
+      href="images/logo/air-logo.svg">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&family=Poppins:wght@600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/reset.css">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Dia</h1>
+  <body class="page__body">
+    <header class="page_header header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__actions">
+            <a href="#" class="logo">
+              <img
+                src="images/logo/air-logo.svg"
+                alt="Dia"
+                class="logo__img">
```

Comment:
```suggestion
                class="logo__img"
              >
```
Треба всюди виправити форматування